### ISSUE TITLE
tensorflow mixes up labels on channels with multiple units

### ISSUE THREAD

**Issue by sg-s** [sg-s/crabsort#74]
if a channel can have unit A, B and X (noise), but if the training set only contains A and X, X is assigned the label 2 (when it should be 3). this means that when predictions are returned, the label of 2 is interpreted as B (when it really means X).

**Comment by sg-s** [comment]
is this still an issue?

**Comment by sg-s** [comment]
n/a
